EmbedTool/EmbedTool.cs

```diff
@@ -10,8 +10,28 @@ namespace EmbedsOnFly
 {
     public class EmbedTool : ModuleBase
     {
-        EmbedBuilder basic;
-        EmbedFooterBuilder footer;
+        private EmbedBuilder basic;
+        private EmbedFooterBuilder footer;
+
+        private readonly IDictionary<string, Color> _paletteMap = new Dictionary<string, Color>()
```

Comment:
I'd make this static to avoid re-instantiation of the dictionary on each instantiation of `EmbedTool`.

Reply:
Agreed.